app/Events/NetworkDataUpdatedEvent.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace App\Events;
+
+class NetworkDataUpdatedEvent
```

Comment:
Is this meant to be empty?

Reply:
Yes - in this instance we want the underlying jobs to query the data for themselves and only process what they need, rather than having the entire database full of aircraft. The event is just to signify "ave at it".